kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -24,6 +24,8 @@
 namespace Kratos {
 namespace Testing {
 
+    const double epsilon = std::numeric_limits<double>::epsilon()*1e3;
```

Comment:
I wouldn't add a common `epsilon` for the entire `Testing` namespace (at least not in here).

Reply:
Ok, I will define it locally at each test